autoload/ensime.vim

```diff
@@ -43,14 +43,6 @@ function! ensime#au_buf_leave(filename) abort
     return s:call_plugin('au_buf_leave', [a:filename])
 endfunction
 
-function! ensime#au_cursor_hold(filename) abort
```

Comment:
I think we shouldn't discard this implementation totally, we should use timers when `+timers` is present and fallback to this implementation otherwise.

Maybe we can encapsulate the scheduling implementation details inside `Editor` class as much as possible.